examples/config/config.example.yaml

```diff
@@ -51,7 +51,6 @@ authenticate_service_url: https://authenticate.localhost.pomerium.io
 
 ## GOOGLE
 # idp_provider: "google"
-# idp_provider_url: "https://accounts.google.com" # optional for google
```

Comment:
Could we replace this reference with another provider this would be relevant for? Maybe azure as you did below in the docs?

Reply:
Look at the entire file, and not just the diff snippet.